charts/telepresence/templates/deployment.yaml

```diff
@@ -94,6 +94,8 @@ spec:
             containerPort: 8081
           - name: https
             containerPort: 8443
+          - name: prometheus
```

Comment:
I would like this to be configurable and have Prometheus disabled by default. We don't want to add extra services unless there's a need for them. It may conflict with Istio rules etc. in the cluster that must be configured.